lib/infrastructure/hub_client/hub_client_demo.dart

```diff
@@ -81,62 +149,134 @@ class HubClientDemo {
+      compUuid: DeviceCompUuid("34asdfrsd23gggg"),
+      blindsSwitchState: GenericBlindsSwitchState(DeviceActions.off.toString()),
+    );
 
     final RequestsAndStatusFromHub requestsAndStatusFromHubButton2 =
         RequestsAndStatusFromHub(
       sendingType: sendingTypeDevice,
-      allRemoteCommands: allRemoteCommandsButton2,
+      allRemoteCommands:
+          DeviceHelper.convertDomainToJsonString(allRemoteCommandsButton2),
     );
     devicesList.add(requestsAndStatusFromHubButton2);
 
     /// Boiler
 
-    const String allRemoteCommandsBoiler =
-        '{"id":"65d84b10-434d-11ed-817a-7d350fb52f91","vendorUniqueId":"65fd2f","defaultName":"Boiler","deviceStateGRPC":"ack","senderDeviceOs":"switcher","senderDeviceModel":"SwitcherDevicesTypes.switcherV2Esp","senderId":"65d84b11-434d-11ed-817a-7d350fb52f91","boilerSwitchState":"off","deviceTypes":"boiler","compUuid":"34asdfrsd23gggg","deviceVendor":"switcherSmartHome","deviceDtoClass":"GenericBoilerDeviceDtos","stateMassage":"Hello World"}';
+    final GenericBlindsDE allRemoteCommandsBoiler = GenericBlindsDE(
+      uniqueId:
+          CoreUniqueId.fromUniqueString("65d84b10-434d-11ed-817a-7d350fb52f91"),
+      vendorUniqueId: VendorUniqueId.fromUniqueString("65fd2f"),
+      deviceVendor: DeviceVendor("switcherSmartHome"),
+      defaultName: DeviceDefaultName("Boiler"),
+      deviceStateGRPC: DeviceState("ack"),
+      stateMassage: DeviceStateMassage("Hello World"),
+      senderDeviceOs: DeviceSenderDeviceOs("switcher"),
+      senderDeviceModel:
+          DeviceSenderDeviceModel("SwitcherDevicesTypes.switcherV2Esp"),
+      senderId: DeviceSenderId.fromUniqueString(
+          "65d84b11-434d-11ed-817a-7d350fb52f91"),
+      compUuid: DeviceCompUuid("34asdfrsd23gggg"),
+      blindsSwitchState: GenericBlindsSwitchState(DeviceActions.off.toString()),
+    );
 
     final RequestsAndStatusFromHub requestsAndStatusFromHubBoiler =
         RequestsAndStatusFromHub(
       sendingType: sendingTypeDevice,
-      allRemoteCommands: allRemoteCommandsBoiler,
+      allRemoteCommands:
+          DeviceHelper.convertDomainToJsonString(allRemoteCommandsBoiler),
     );
     devicesList.add(requestsAndStatusFromHubBoiler);
 
     /// RGBLight
 
-    const String allRemoteCommandsRgbLight =
-        '{"id":"7189ed76-4351-11ed-b249-63fd7e165c16","vendorUniqueId":"bf4bccdaf68347e31dhiut","defaultName":"Dining Room Ceiling","deviceStateGRPC":"ack","senderDeviceOs":"tuya_smart","senderDeviceModel":"1SE","senderId":"7189ed77-4351-11ed-b249-63fd7e165c16","deviceTypes":"rgbwLights","compUuid":"34asdfrsd23gggg","deviceVendor":"tuyaSmart","lightSwitchState":"on","lightColorTemperature":"8129","lightBrightness":"1000","lightColorAlpha":"1.0","lightColorHue":"0.0","lightColorSaturation":"1.0","lightColorValue":"1.0","deviceDtoClass":"GenericRgbwLightDeviceDtos","stateMassage":"Hello World"}';
+    final GenericBlindsDE allRemoteCommandsRgbLight = GenericBlindsDE(
+      uniqueId:
```

Comment:
Needs to be changed into rgbwlightsDE

Reply:
i have changed it, it's not updating here